src/pages/Heightmap.vue

```diff
@@ -539,19 +539,7 @@ export default class PageHeightmap extends Mixins(BaseMixin, ControlMixin) {
                 calculable: true,
                 dimension: 2,
                 inRange: {
-                    color: [
-                        '#313695',
-                        '#4575b4',
-                        '#74add1',
-                        '#abd9e9',
-                        '#e0f3f8',
-                        '#ffffbf',
-                        '#fee090',
-                        '#fdae61',
-                        '#f46d43',
-                        '#d73027',
-                        '#a50026',
-                    ],
+                    color: this.$store.getters['gui/heightmap/getActiveColorSchemeList'],
```

Comment:
pls use a separate getter and don't use the store getter inside of this huge object